Library/Tracking/Koala.swift

```diff
@@ -355,15 +356,17 @@ public final class Koala {
 
   public init(
     bundle: NSBundleType = Bundle.main,
-    client: TrackingClientType,
+    dataLakeClient: TrackingClientType = TrackingClient(.dataLake), // TODO: Remove default value
```

Comment:
Also to reduce the amount of line changes in this PR I'm passing in a default value here. We can tackle this separately.